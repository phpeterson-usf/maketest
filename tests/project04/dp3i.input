-h $(repo_dir)/dp3i.hex $(TESTS_DIR)/dp3i.s
