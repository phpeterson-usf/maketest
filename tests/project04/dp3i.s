main:
    add r1, r2, #99
    bx lr
