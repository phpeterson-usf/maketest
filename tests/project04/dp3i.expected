v2.0 raw
e2821063
e12fff1e
